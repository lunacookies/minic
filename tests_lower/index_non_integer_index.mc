func main {
	x := 5
	a := [1, 2]
	return a[&x]
}
