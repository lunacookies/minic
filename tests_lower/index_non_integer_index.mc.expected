func main
	var x i64
	var a [2]i64
	{
		set x = 5
		set a = [
			1,
			2,
		]
		return <missing>
	}
tests_lower:41..42: error: index is non-integer type “*i64”
